.appveyor.yml

```diff
@@ -95,6 +95,12 @@ install:
   # pytest-cov>=2.3.1 due to https://github.com/pytest-dev/pytest-cov/issues/124
   - pip install -q pytest "pytest-cov>=2.3.1" pytest-rerunfailures pytest-timeout
 
+  # Apply patch to `subprocess` on Python versions > 2 and < 3.6.3
+  # https://github.com/matplotlib/matplotlib/issues/9176
+  - python -c "import sys; sys.exit((2,) > sys.version_info > (3,6,3))" && (
```

Comment:
Shouldn't this be `sys.exit((3,6,3) > sys.version_info > (2,))` ?

Reply:
Yes, I wrongly negated the `(2,) < sys.version_info < (3,6,3)` expression.